### ISSUE TITLE
apply pose relative to user defined object

### ISSUE THREAD

**Issue by friendshipmachine** [krathjen/studiolibrary#401]
Currently, the pose is applied to the world as it's being recorded from previous scenes in that location.  However, to apply the pose on the existing scene relative to a joint (or control) would be a fantastic option to have.

**Comment by thoolaeff** [comment]
yes this would be amazing, motion builder's pose tool does this and it is fundamental to our workflows in motion builder
